docs/develop/go/test-suites.mdx

```diff
@@ -30,7 +31,9 @@ In the context of Temporal, you can create these types of automated tests:
   - Running Activities with mocked Context and other SDK imports (and usually network requests).
   - Running Workers with mock Activities, and using a Client to start Workflows.
   - Running Workflows with mocked SDK imports.
-- **Unit:** Running a piece of Workflow or Activity code (a function or method) and mocking any code it calls.
+- **Unit:**
+  - Running a piece of Workflow or Activity code (a function or method) and mocking any code it calls.
+  - Running a piece of Nexus operation and mocking the handler.
```

Comment:
I would remove this because this isn't testing the nexus operation. It's testing a piece of Workflow code that executes an operation, which is included above.